src/integration-test/java/uk/gov/hmcts/juror/api/AbstractControllerIntegrationTest.java

```diff
@@ -166,6 +167,12 @@ public ControllerTestResponse<T> printResponse() {
                 return this;
             }
 
+            @SneakyThrows
+            public ControllerTestResponse<T> printResponseBodyAsJson() {
+                System.out.println(new ObjectMapper().findAndRegisterModules().writeValueAsString(body));
```

Comment:
Is the print statement intentional?

Reply:
Yes :)

This method is a support method to add logs to your tests when you are creating them.